mfr/extensions/tabular/libs/stdlib_tools.py

```diff
@@ -30,3 +34,21 @@ def csv_stdlib(fp):
         })
     rows = [row for row in reader]
     return columns, rows
+
+
+def detect_quoted_fields(dialect, data):
```

Comment:
Cannot take advantage of quoting options. Default is `QUOTE_MINIMAL`, which should be (and is when `dialect.quotechar` is set properly) handling the management of delimiters inside of fields. 

The issue is that the sniffer detects the `dialect.quotechar` incorrectly for large lists of non-quotechar-wrapped items inside of the actual quotechar. I tried numerous test files, including ones in which all quotation marks, single or double, were escaped. 

Regex use is necessary to catch all possible formatting versions of this type of bug.